site/en/docs/android/trusted-web-activity/quick-start/index.md

```diff
@@ -114,129 +112,19 @@ use PWA Builder to create an Android App that opens your PWA in
-
-#### Jekyll based websites
-
-If your website is generated by Jekyll (such as GitHub Pages), you'll need to add a line of
-configuration so that the `.well-known` directory is included in the output.
-GitHub help has [more information on this topic](https://help.github.com/en/articles/files-that-start-with-an-underscore-are-missing).
-Create a file called `_config.yml` at the root of your site (or add to it if it already exists) and
-enter:
-
-```yml
-# Folders with dotfiles are ignored by default.
-include: [.well-known]
-```
-
-### Adding more keys {: #adding-more-keys }
-
-A Digital Asset Link file can contain more than one app, and for each app, it can contain more than
-one key.
-For example, to add a second key, just use the
-[Asset Link Tool](https://play.google.com/store/apps/details?id=dev.conn.assetlinkstool) to
-determine the key and add it as a second entry to the `sha256_cert_fingerprints` field.
-The code in Chrome that parses this JSON is quite strict, so make sure you don't accidentally add an
-extra comma at the end of the list.
-
-```json
-[{
-  "relation": ["delegate_permission/common.handle_all_urls"],
-  "target": {
-    "namespace": "android_app",
-    "package_name": "com.appspot.pwa_directory",
-    "sha256_cert_fingerprints": [
-      "FA:2A:03:CB:38:9C:F3:BE:28:E3:CA:7F:DA:2E:FA:4F:4A:96:F3:BC:45:2C:08:A2:16:A1:5D:FD:AB:46:BC:9D"
-    ]
-  }
-},{
-  "relation": ["delegate_permission/common.handle_all_urls"],
-  "target": {
-    "namespace": "android_app",
-    "package_name": "com.appspot.pwa_directory",
-    "sha256_cert_fingerprints": [
-      "4F:FF:49:FF:C6:1A:22:E3:BB:6F:E6:E1:E6:5B:40:17:55:C0:A9:F9:02:D9:BF:28:38:0B:AE:A7:46:A0:61:8C"
-    ]
-  }
-}]
-```
-
-## Troubleshooting {: trouble-shooting }
-
-### Viewing relevant logs {: viewing-relevant-logs }
-Chrome logs the reason that Digital Asset Links verification fails and you can view the logs on an
-Android device with `adb logcat`.
-If you're developing on Linux/Mac, you can see the relevant logs from a connected device
-with:
-
-```shell
-> adb logcat -v brief | grep -e OriginVerifier -e digital_asset_links
-```
-
-For example, if you see the message `Statement failure matching fingerprint.`, you should use the
-Asset Link Tool to see your app's signature and make sure it matches that in your `assetlinks.json`
-file (Be wary of confusing your debug and release keys. Look at the
-[A note on signing keys](#a-note-on-signing-keys) section.)
-
-### Checking your browser {: #checking-your-browser }
+After creating your `assetlinks.json` file, upload it to your website at `.well-known/assetlinks.json` relative to the root) so that your app can be verified properly by the browser. Check out a [deep dive on Digital Asset Links](/docs/android/trusted-web-activity/android-for-web-devs#digital-asset-links) for more information on how it relates to your signing key.
+## Checking your browser {: #checking-your-browser }
```

Comment:
nit: I think there should be a newline before headers in markdown.

Reply:
Thanks, Peter. Good catch!